manifests/kubernetes/kubevirt-tekton-tasks-kubernetes.yaml

```diff
@@ -191,7 +193,7 @@ spec:
       description: The namespace of a VM that was created.
   steps:
     - name: createvm
-      image: "quay.io/kubevirt/tekton-task-create-vm:v0.14.0"
+      image: "quay.io/kubevirt/tekton-tasks:v0.14.0"
       command:
         - create-vm
```

Comment:
Not using `entrypoint`?

Reply:
Some tasks does not need a user modifications. 
Another problem is, that the entrypoint is doing some nasty things with quotation marks and e.g. modify-template task has problem with it.